async_test.go

```diff
@@ -74,6 +85,9 @@ func TestCreateValidDownloadJobs(t *testing.T) {
 }
 
 func TestCreateInvalidDownloadJobs(t *testing.T) {
+	_, cancel := context.WithTimeout(context.Background(), taskTimeout)
+	defer cancel()
```

Comment:
I think that this can't really do anything because nothing is listening for the context. should either use the context or remove

Reply:
Removed.